src/w3d/renderer/render2dsentence.cpp

```diff
@@ -207,8 +306,149 @@ void FontCharsClass::Update_Current_Buffer(int char_width)
     }
 }
 
+bool FontCharsClass::Locate_Font(const char *font_name, StringClass &font_file_path)
```

Comment:
I suggest to make this function call 2 sub functions

Locate_Freetype_Font
Locate_Windows_Font

This way it becomes slimmer.

```c++
void FontCharsClass::Locate_Font(...)
{
#if defined(BUILD_WITH_FREETYPE)
  Locate_Freetype_Font(...);
#else defined(PLATFORM_WINDOWS)
  Locate_Windows_Font(...)
#endif
}
```